src/modules/accounting/accounting.py

```diff
@@ -224,11 +225,28 @@ def get_rebase_after_report(self, blockstamp: ReferenceBlockStamp, cl_only=False
 
         return LidoReportRebase(*result)
 
+    def _get_slots_elapsed_from_last_report(self, blockstamp: ReferenceBlockStamp):
+        """If no report was finalized return slots elapsed from initial epoch from contract"""
+        chain_conf = self.get_chain_config(blockstamp)
+        frame_config = self.get_frame_config(blockstamp)
+
+        last_ref_slot = self._get_processing_ref_slot(blockstamp)
+
+        if last_ref_slot:
+            slots_elapsed = blockstamp.ref_slot - last_ref_slot
+        else:
+            slots_elapsed = blockstamp.ref_slot - frame_config.initial_epoch * chain_conf.slots_per_epoch
+
+        return slots_elapsed
+
+    def _get_processing_ref_slot(self, blockstamp: BlockStamp):
+        return self.report_contract.functions.getLastProcessingRefSlot().call(block_identifier=blockstamp.block_hash)
```

Comment:
it's also used in bunker and save_border services, better to have is one place

Reply:
fixed